bin/compile

```diff
@@ -440,7 +445,14 @@ case "${TOOL}" in
     gomodules)
         pkgs=${GO_INSTALL_PACKAGE_SPEC:-$(awk '{ if ($1 == "//" && $2 == "+heroku" && $3 == "install" ) { print substr($0, index($0,$4)); exit }}' ${goMOD})}
         if [ -z "${pkgs}" ]; then
-            pkgs="default"
+            if [ -d "${build}/cmd" ]; then
```

Comment:
Can we add `mcount`s to both of these branches so we can see how often each is used?

Reply:
Yes I will